tasks/release.py

```diff
@@ -909,6 +919,34 @@ def chase_release_managers(_, version):
         client.chat_postMessage(channel=channel, text=message)
 
 
+@task
+def chase_for_qa_cards(_, version):
+    from collections import defaultdict
+
+    from slack_sdk import WebClient
+
+    cards = list_not_closed_qa_cards(version)
+    if len(cards) == 0:
+        print("No QA cards to chase", file=sys.stderr)
+        return
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you instead use something like:
```suggestion
        Exit(color_message("No QA cards to chase", Color.RED), code=1)
```
With
```
from invoke.exceptions import Exit
from tasks.libs.common.color import Color, color_message
```

Reply:
`Exit` is an exception so I'm not sure about `Exit` itself. I'll propose something